R/create_title.R

```diff
@@ -5,10 +5,18 @@
 #' @return NULL
 #' @export
 #'
-#' @examples create_title(
-#'   office = "SEFSC", species = "Red Snapper", region = "South Atlantic",
-#'   year = 2024, type = "SAR", spp_latin = "Lutjanus campechanus"
+#' @examples
```

Comment:
another situation like `create_citation` where the example could actually run, you can name the fxn, then cat(name)

Reply:
👍 Edited as per your suggestion.